andela/persephone-ah-frontend
# #167623666 Create a loading indicator component

#### What does this PR do?
Adds loading indicator component

#### Description of Task to be completed?
 > - Create a reusable component that indicates that a request is pending

#### How should this be manually tested?
> - Clone the repo
> - run `npm install` and `npm test'
> - The tests should be passing

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#167623666](https://www.pivotaltracker.com/story/show/167623666)

#### Screenshots (if appropriate)
![loadingindicator](https://user-images.githubusercontent.com/33371467/62242889-18ade900-b3d4-11e9-8dcc-fbaa8f983e4d.gif)

#### Questions:
NA
